docs/installation/production/additional/official_docker.rst

```diff
@@ -345,6 +344,12 @@ Here is an example of a dockerfile from a tethys app:
     ENV APP_DB_PASSWORD ${TETHYS_DB_PASSWORD}
     ENV CONDORPY_HOME ${TETHYS_HOME}/tethys
 
+    ##################################
+    # PRODUCTION ENVIRONMENT VARIABLES
+    ##################################
+    ENV ASGI_PROCESSES 1
+    ENV CHANNEL_LAYER "channels_redis.core.RedisChannelLayer"
```

Comment:
Don't you need to specify the host and path of the Redis service?

Reply:
Yes you do, that's a few lines below on the salt part.

Reply:
You mean in the django_channels_layer.rst doc? So there's no way to enable the Redis channel layer without extending the Dockerfile and building your own?